app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1972,6 +1965,18 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return false;
     }
 
+    @Override
+    public boolean onPrepareOptionsMenu(Menu menu) {
+        Log.d("nitish","status"+networkStatus);
```

Comment:
Remove this. @vishwesh3 already told you about this. https://github.com/zulip/zulip-android/pull/316#discussion_r99484688